x-pack/solutions/security/plugins/security_solution/public/app/home/index.tsx

```diff
@@ -27,18 +27,21 @@ import { TopValuesPopover } from '../components/top_values_popover/top_values_po
 import { AssistantOverlay } from '../../assistant/overlay';
 import { useInitSourcerer } from '../../sourcerer/containers/use_init_sourcerer';
 import { useInitDataViewManager } from '../../data_view_manager/hooks/use_init_data_view_manager';
+import { useRestoreDataViewManagerStateFromURL } from '../../data_view_manager/hooks/use_sync_url_state';
 
 interface HomePageProps {
   children: React.ReactNode;
 }
 
 const HomePageComponent: React.FC<HomePageProps> = ({ children }) => {
   const { pathname } = useLocation();
-  const { browserFields } = useInitSourcerer(getScopeFromPath(pathname));
+  const sourcererScope = getScopeFromPath(pathname);
+  const { browserFields } = useInitSourcerer(sourcererScope);
```

Comment:
We'll want to replace this with `useBrowserFields` when this initialization logic can be removed